howto/port/landing.md

```diff
@@ -0,0 +1,7 @@
+When porting an Android app to Anbox Cloud (usually in the form of an APK), there are a few issues that might cause your app to not function properly:
+
+* Missing dependencies, most importantly to Google Play services. Google Play services are not supported by Anbox Cloud, and apps that require Google Play services can therefore not be ported to Anbox Cloud.
+* Missing runtime permissions. See [Grant runtime permissions](tbd) for instructions on how to grant the required permissions.
```

Comment:
Those `tbd`s are placeholders and will be replaced with the real URL when we publish these?

Reply:
Yes, exactly. We don't know the URL before publishing, unfortunately. :(

Reply:
I see.